src/main/java/org/maxgamer/quickshop/shop/DisplayItem.java

```diff
@@ -136,6 +136,7 @@ public static boolean checkIsTargetShopDisplay(@NotNull final ItemStack itemStac
         }
         String defaultMark = ShopProtectionFlag.getDefaultMark();
         String shopLocation = shop.getLocation().toString();
+        String attachedShopLocation = (!shop.isRealDouble()) ? "null" : (shop.getAttachedShop() == null ? "null" : shop.getAttachedShop().getLocation().toString());
```

Comment:
Please convert to an if statement, ternary expression will make it unreadable

Reply:
You're very right, that was a quick fix since I already had a single ternary and I just slapped another one on.